### DISCUSSION TITLE
Questions about code in `server.js`

### DISCUSSION THREAD

**Discussion by hens-virtuella-varld** [data-interaction-fed24/server#6] [Q&A] [answered]
https://github.com/data-interaction-fed24/server/blob/main/server.js
```
    const queryText = 'INSERT INTO my_table(name) VALUES($1) RETURNING *';
    const values = [name];
```

1.
`values = [name]`, this `values` is for `VALUES($1) `. Is it correct?

2.
The `name` in `my_table(name)`, where do we get the value?

3.
>`RETURNING *`
Does it mean to return all the data?
This is a post request so I don't think so.

**Comment by Liam-hi** [comment, marked as answer]
1. Correct, $1 indicate where the value 'name' should be inserted.

2. Because you are referring to the query we use for POST, I'm assuming you asking for how we can add data?
- open Postman
- select POST
- Add the link http://localhost:3001/api/my_table
- Add following to the Body 
```
 {
  "name": "Regan"
}
```
This will add the name Regan to the database.

To see the data, we use the GET method. 

3. We want to see if their was any error wen we added the data, that's why we want to return the data. We can remove the * when we do experiments in Postman to see effects.
